.circleci/config.yml

```diff
@@ -39,40 +43,39 @@ jobs:
       - run:
           name: Install deps
           command: |
-            npm install
+            yarn install
             sudo apt-get install poppler-utils
       - run:
           name: List deps
           command: |
-            npm ls
+            yarn list
             pdftops -v
       - save_cache:
           paths:
             - node_modules
           key: v{{ .Environment.CIRCLE_CACHE_VERSION }}-deps-{{ .Branch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "package.json" }}
       - run:
           name: Test
-          command: npm test
+          command: yarn run test
       - run:
           name: Coverage
-          command: npm run coverage
+          command: yarn run coverage
       - store_artifacts:
           path: build
 
   test-node-v6:
     <<: *base
     docker:
-      - image: circleci/node:6.10.3-browsers
+      - image: circleci/node:6-browsers
 
   test-node-v8:
     <<: *base
     docker:
-      - image: circleci/node:8.4.0-browsers
+      - image: circleci/node:8-browsers
```

Comment:
Why?

Reply:
I prefer locking down the image versions. I don't think this project is likely to break from one minor release of node.js to another.

Reply:
Okay, I have made this change. How's it look?